### PR TITLE
Add sentry error tracker

### PR DESCRIPTION
**Pull request by csaba-nagy**

### CODE REVIEW

File: packages/server/src/app/create-app.ts [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
+    dsn: process.env.SENTRY_DSN,
+    integrations: [
+      new SentryIntegrations.Http({ tracing: true }),
+      new TracingIntegrations.Express({ app }),
+    ],
+    // eslint-disable-next-line object-shorthand
+    tracesSampleRate: tracesSampleRate,
+  })
+
+  app.use(SentryHandlers.requestHandler() as express.RequestHandler)
```
Comment (nandordudas) [inline packages/server/src/app/create-app.ts:23]: Just use `RequestHandler` as type.

File: packages/server/src/routes/error/error.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express'
+import { routerOptions } from '@routes/config'
+import { AllowedRequestMethod, RoutePath } from '@types'
+
+export const router = Router(routerOptions)
+
+const message = 'Api Error'
+
+router[AllowedRequestMethod.GET](RoutePath.Error, (): void => {
```
Comment (nandordudas) [inline packages/server/src/routes/error/error.ts:9]: Explicit return type not necessary.

File: packages/server/src/routes/index.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import type { Router } from 'express'
+import { router as errorRouter } from '././error/error'
```
Comment (nandordudas) [inline packages/server/src/routes/index.ts:2]: What it mean `././`, is this working properly? 🤨

**Review by nandordudas — CHANGES_REQUESTED** [review]

File: packages/server/src/routes/index.ts (hunk 3 above)
Comment (csaba-nagy) [reply packages/server/src/routes/index.ts:2]: It's a typo, but surprisingly it works... I think Copilot did a funny sabotage here...👀 
On the other hand, I named the file in a wrong way. Both of them will be fixed.

**Pull request merged**